packages/datagrid/src/index.js

```diff
@@ -0,0 +1,3 @@
+import DataGrid from './containers/datagrid.connect';
```

Comment:
`import Datagrid from './containers';`
of your `src/containers/index.js` is useless